docs/resources/ocean_aks_virtual_node_group.md

```diff
@@ -99,3 +99,31 @@ The following arguments are supported:
         * `key` - (Optional) Tag Key for Vms in the cluster.
         * `value` - (Optional) Tag Value for VMs in the cluster.
   * `max_pods` - (Optional) The maximum number of pods per node in an AKS cluster.
+<a id="update-policy"></a>
+## Update Policy
+
+* `update_policy` - (Optional)
+  * `should_roll` - (Required) If set to true along with the vng update, roll will be triggered.
+  * `conditioned_roll` - (Optional, Default: false) Spot will perform a cluster Roll in accordance with a relevant modification of the cluster’s settings. When set to true , only specific changes in the cluster’s configuration will trigger a cluster roll (such as availability_zones, max_pods_per_node, enable_node_public_ip, os_disk_size_gb, os_disk_type, os_sku, kubernetes_version, vnet_subnet_ids, pod_subnet_ids, labels, taints and tags).
+  * `roll_config` - (Required) While used, you can control whether the group should perform a deployment after an update to the configuration.
```

Comment:
in the code it is defined as Optional, please re-validate

Reply:
Updated to optional.